bftengine/tests/simpleStorage/FileStorage.cpp

```diff
@@ -43,6 +43,9 @@ FileStorage::FileStorage(Logger &logger, const string &fileName) : logger_(logge
 }
 
 FileStorage::~FileStorage() {
+  if (dontLoadStorageOnStartup) {
```

Comment:
Do we need FileStorage at all?

Reply:
I don't know, it is there so I have handled it too.

Reply:
ask @yuliasherman , we can probably through it away